src/main/kotlin/com/github/pool_party/pull_party_bot/command/Messages.kt

```diff
@@ -10,6 +10,8 @@ package com.github.pool_party.pull_party_bot.commands
 // /help command suggestion
 // """
 
+// TODO messages for alias commands
```

Comment:
I believe it would be nice to have messages before the merge

Reply:
I would let **you** do this task, u r the best in  this field. And you can look at the command interface, as well. U can make more revisions on some kind of user experience, idk.

Btw, I am going to fix linter & code factor warnings later this evening